src/pages/Home.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { useState, useEffect } from "react";
+import axios from "axios";
+import { Pagination, Paper, Toolbar, Typography, AppBar } from "@mui/material";
+import NFTCard from "../components/card/NFTCard";
+
+const NFTsPerPage = 5;
+
+const Home = () => {
+  const [nfts, setNFTs] = useState<any[]>([]);
+  const [loading, setLoading] = useState<boolean>(true);
+  const [error, setError] = useState<string | null>(null);
+  const [page, setPage] = useState(1);
+
+  const API_KEY = "5402e464f97c44968342dd3057b4a262";
```

Comment:
api keys always go to  .env file . you are never supposed to publicly show them, they should be private